docs/UserGuide.md

```diff
@@ -165,7 +167,16 @@ Shows a message explaining how to access the help page.
 
 ![help message](images/helpMessage.png)
 
-Format: `help`
+Format: `help [COMMAND]`
+
+* Enter optional parameter `COMMAND` to view the usage for that command.
+![meet Help Message](images/meetHelpMessage.png)
+* Leave parameter blank to view message explaining how to access help page and the usage of all commands.
```

Comment:
Nice explanation. It would be better to use some other word than `parameter`. Maybe `Entering only the help will allow you to`

Reply:
Good suggestion! Will be changing it